micrometer-tracing/src/main/java/io/micrometer/tracing/contextpropagation/BaggageToPropagate.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.tracing.contextpropagation;
+
+import java.util.Map;
+
+/**
+ * Wrapper around baggage to propagate.
+ *
+ * @since 1.2.2
+ * @author Marcin Grzejszczak
+ */
+public class BaggageToPropagate {
+
+    private final Map<String, String> baggage;
+
+    public BaggageToPropagate(Map<String, String> baggage) {
+        this.baggage = baggage;
```

Comment:
Should this be a reference assignment or a copy? Please see my other comment in `BaggageThreadLocalAccessor`.